File: app/src/main/java/cn/intret/app/picgo/ui/event/CurrentImageChangeMessage.java
package cn.intret.app.picgo.ui.event;

/**
 * 图片查看器当前显示图片发生变化
 */

public class CurrentImageChangeMessage {
    int mPosition;

    public int getPosition() {
        return mPosition;
    }

    public CurrentImageChangeMessage setPosition(int position) {
        mPosition = position;
        return this;
    }

    @Override
    public String toString() {
        return "CurrentImageChangeMessage{" +
                "mPosition=" + mPosition +
                '}';
    }
}

File: app/src/main/java/cn/intret/app/picgo/screens/event/CurrentImageChangeMessage.kt
package cn.intret.app.picgo.screens.event

/**
 * 图片查看器当前显示图片发生变化
 */

class CurrentImageChangeMessage {
    internal var mPosition: Int = 0

    fun getPosition(): Int {
        return mPosition
    }

    fun setPosition(position: Int): CurrentImageChangeMessage {
        mPosition = position
        return this
    }

    override fun toString(): String {
        return "CurrentImageChangeMessage{" +
                "mPosition=" + mPosition +
                '}'.toString()
    }
}
